src/app/public/modules/vertical-tabset/fixtures/vertical-tabset-no-active.component.fixture.html

```diff
@@ -1,4 +1,6 @@
-<sky-vertical-tabset showTabsText='Tab list'>
+<sky-vertical-tabset
+  [maintainTabContent]="maintainTabContent"
+  showTabsText='Tab list'>
```

Comment:
I'm sorry to be very nitpicky.... could we switch this to have the bound property after `showTabsText`? That is just something we do internally and it will keep things standard.

Reply:
Sounds good.  It's nice to be consistent, so I'll fix all these up.